### DISCUSSION TITLE
Suggestion: Energy distribution as individual card

### DISCUSSION THREAD

**Discussion by nagyrobi** [home-assistant/frontend#9725] [Dashboard] [answered]
Please add the Energy distribution window to the cards list as an individual card which we can add freely to any custom view/page. Optionally with a shortcut: when clicking on this card, navigate to the big Energy page.

**Comment by nsmits** [comment, marked as answer]
This is possible: https://www.home-assistant.io/lovelace/energy/

**Reply by nagyrobi** [reply]
Thank you.
